modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/client/extension/ClientExtension.java

```diff
@@ -73,14 +73,9 @@ public Map<String, Object> toJSONMap(String pid) {
 			unmappedProperties.getOrDefault(
 				"webContextPath", "/" + projectName));
 
-		Set<Map.Entry<String, Object>> set = unmappedProperties.entrySet();
-
-		set.forEach(
-			entry -> {
-				if (!pid.contains("CETConfiguration")) {
-					configMap.put(entry.getKey(), entry.getValue());
-				}
-			});
+		if (!Objects.equals("frontend", classification)) {
+			configMap.putAll(unmappedProperties);
+		}
```

Comment:
Any reason we can't always add unmapped properties?

Reply:
hmm, I guess there's not a strong reason.

Reply:
fixed

Reply:
My thought on this one is that it sets me up to try adjusting the CET side of things on portal's side and maybe remove the `typeSettings` concept from the workspace plugin completely, per our conversation last week.